### DISCUSSION TITLE
(Suggestion) Support for newer HUD's

### DISCUSSION THREAD

**Discussion by Big-the-Fat** [andrei-drexler/ironwail#29] [Ideas]
Would it be possible to add the widescreen HUD's found in Arcane Dimensions when run in QS and QSS?  It would be really nice if they could be used as a default HUD for any map.  Also, after playing KexQuake it'd be nice to have the option to use the N64 style HUD along with top-centred messages.  My first exposure to Quake was on the N64 so it'd be a great nostalgia feature. :)

**Comment by andrei-drexler** [comment]
AD's HUDs are coded in client-side QuakeC, which QS doesn't support. I'll look into adding it, but it seems to be a pretty substantial change. I would also like to add some new generic HUD layouts as alternatives to the classic one, though, but they wouldn't be aware of any of the custom items a mod might have. I guess the N64 layout could be one of them, but I'll have to take a look at KexQuake to see what it's like since I only ever played Quake on PC (first time was on a 486 DX4 100 MHz... it didn't run that well!)

**Reply by Immorpher** [reply]
These videos perhaps can help you out.

On the original N64 they moved the HUD info to the bottom corners of the 4:3 screen with some transparency. Armor and health on the left, with ammo and keys on the right: https://youtu.be/AHYIwoc7PCU

KEX Quake keeps it similar, but since its for a 16:9 screen the HUD elements get pushed further apart: https://www.youtube.com/watch?v=3zmHd7hjGDc

I think its pretty good as it keeps the HUD away from the action in the center of the screen.

**Comment by Big-the-Fat** [comment]
Pretty sure it ran better on your 486 than on my N64! 

Thanks for taking a look at the HUD.  I appreciate it. :)
